test/browser/features/support/pages/npsp_batch_data_entry_page.rb

```diff
@@ -1,7 +1,7 @@
 class NPSPBDEPage
   include PageObject
 
-  text_field(:batch_name, index: 1)
+  text_field(:batch_name, id: 'j_id0:j_id18:j_id19:j_id21:j_id22:0:j_id23')
```

Comment:
is this something we can get a named ID or class on instead of the salesforce nonsense?